### PR TITLE
Edit Page - Data of current user added and updated

### PR DESCRIPTION
**Pull request by gid-ctrl**

### CODE REVIEW

File: src/pages/edit/index.js [hunk 1]
```diff
@@ -4,8 +4,40 @@ import TextInput from "../../components/form/textInput";
...
+  const { userId } = useAuth();
+  const [user, setUser] = useState({});
+  const [userInitials, setUserInitials] = useState(``);
+
+  useEffect(() => {
+    async function getUserInfo() {
+      const userInfo = await get(`users/${userId}`);
+      setUser(userInfo.data.user);
+      setUserInitials(getInitailsFromUser(userInfo.data.user));
+      console.log(userInfo);
```
Comment (ample-samples) [inline src/pages/edit/index.js:22]: Don't need this console.log

**Review by ample-samples — CHANGES_REQUESTED** [review]
Works great, just needs a tidy up

**Review by ample-samples — DISMISSED** [review]
LGTM

**Review by dearshrewdwit — DISMISSED** [review]

**Comment by ample-samples** [comment]
Added the ability to display the profiles of other users and conditionally render the edit button

**Review by ample-samples — APPROVED** [review]

**Pull request merged**